.prettierignore

```diff
@@ -0,0 +1,8 @@
+.hars
```

Comment:
added files here that don't need prettier to format, feel free to suggest more. i added the svelte-kit tsconfig because it was formatting differently and committing and i dont think we care about hte formatting inside of that area. and i specifically added the tsconfig because it wasn't behaving.